app/src/main/java/com/example/b07group7project/shopper_view_previous_orders/getPreviousOrders.java

```diff
@@ -0,0 +1,7 @@
+package com.example.b07group7project.shopper_view_previous_orders;
+
+import com.example.b07group7project.Customer;
+
+public interface getPreviousOrders {
+    void getPreviousOrders(Customer c);
+}
```

Comment:
Refactor so you pass Oncomplete, so it should be something like 
void getCart(OnComplete<PreviousOrder> onComplete);